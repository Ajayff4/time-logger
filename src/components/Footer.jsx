import React from 'react'

export default function Footer() {
    return (
        <div className="footer">
            <center>
                <h1>Copyright &copy; 2020 | Ajay Agrawal</h1>
            </center>
        </div>        
    )
}