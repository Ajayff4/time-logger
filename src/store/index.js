import { createStore, applyMiddleware } from 'redux'
import rootReducer from '../reducers/index'
//import logger from 'redux-logger'
import { composeWithDevTools } from 'redux-devtools-extension'
import rootSaga from '../saga'
import createSagaMiddleware from 'redux-saga'

const sagaMiddleware = createSagaMiddleware()
const store = createStore(
    rootReducer,
    composeWithDevTools(
        applyMiddleware(sagaMiddleware))
)

sagaMiddleware.run(rootSaga)

export default store