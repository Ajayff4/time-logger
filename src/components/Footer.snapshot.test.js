import React from 'react'
import renderer from 'react-test-renderer'
import Footer from './Footer'
import { Provider } from 'react-redux';
import store from '../store';
import { BrowserRouter as Router } from 'react-router-dom';

describe('Footer Snapshot Testing...', () => {
    it('renders component correctly', () => {
        const tree = renderer.create(<Provider store={store}><Router><Footer /></Router></Provider> ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});