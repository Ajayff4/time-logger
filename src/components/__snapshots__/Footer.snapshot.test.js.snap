// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshot Testing... renders component correctly 1`] = `
<footer
  id="footerComponent"
>
  <div
    className="footer"
    id="footer"
  >
    <center>
      <h1
        data-test="footerText"
      >
        Copyright Â© 2020 | Ajay Agrawal
      </h1>
    </center>
  </div>
</footer>
`;
